### PR TITLE
Self-hosting not working on reverse proxy with port =! 5701

### PR DESCRIPTION
**Pull request by jonashendrickx**

### CODE REVIEW

**Review by Tyrrrz — DISMISSED** [review]

File: self-host/traefik/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    image: ghcr.io/passwordless/passwordless-self-host:latest
+    volumes:
+      - /Users/jonashendrickx/src/bitwarden/passwordless-server/self-host/traefik/pwdcache:/etc/bitwarden_passwordless
+    restart: unless-stopped
+    environment:
+      # The port to which our app will listen publicly, necessary for non-request contexts.
+      BWP_PORT: 80
+      BWP_DOMAIN: passwordless.docker.localhost
+    labels:
+      - "traefik.http.routers.passwordless.rule=Host(`passwordless.docker.localhost`)"
```
Comment (Tyrrrz) [inline self-host/traefik/docker-compose.yml:29]: Is this file included as an example or is it used for something?

Comment (jonashendrickx) [reply self-host/traefik/docker-compose.yml]: Yeah, it's an example config for use with Traefik as a reverse proxy.

File: self-host/traefik/simple-example/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+version: '3'
```
Comment (Tyrrrz) [inline self-host/traefik/simple-example/docker-compose.yml:1]: What about `self-host/examples/traefik/docker-compose.yml` instead?

**Review by Tyrrrz — DISMISSED** [review]

File: self-host/traefik/simple-example/docker-compose.yml (hunk 2 above)
Comment (jonashendrickx) [reply self-host/traefik/simple-example/docker-compose.yml]: Moved it

**Review by Tyrrrz — APPROVED** [review]

**Pull request merged**